packages/orchestration/src/util/filtering.ts

```diff
@@ -65,14 +65,16 @@ export function buildAzureContentSafetyFilter<T extends 'input' | 'output'>(
 }
 
 /**
- * Convenience function to build Llama guard filter.
+ * Convenience function to build Llama Guard 3 8B filter.
+ * @param type - Type of the filter, either 'input' or 'output'.
```

Comment:
```suggestion
 * @param type - Type of the filter, either `input` or `output`.
```

Looks better in the IDE